docs/user/introduction.rst

```diff
@@ -0,0 +1,28 @@
+Introduction
+============
+
+This will walk you through what quicknav is and some of what it can do.
+
+What is Quicknav
+----------------
+
+Quicknav is a command line tool that allows you to easily jump to specific locations in your
+filesystem from a simple command. Quicknav is built for terminal navigation on Unix operating systems.
+
+Why Quicknav
+------------
+
+Quicknav allows you to easily set shortcuts to locations in your filesystem. It is made to speed up
+your terminal navigation to commonly accessed places without cluttering your terminal with aliases.
+
+Basic Examples
+--------------
+
+.. code:: bash
+
+    # easily navigate to shortcuts that were set in the config
+    $ nav rs     # go to rust projects folder
+    $ nav py     # go to python projects folder
+    $ nav js     # go to javascript projects folder
+
+Ready to get started with quicknav, head over to the getting started page.
```

Comment:
```suggestion
Ready to get started with quciknav? Head over to the `getting started <https://quicknav.readthedocs.io/en/latest/user/install.html>`_ page!
```

Reply:
I think giving them a little direction here could be helpful.

Reply:
I implemented the same thing just in a slightly different way so that its not tided to the latest docs and can be used on different versions or something like development docs, etc.